keyboards/z150_blackheart/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define ______ KC_TRNS
```

Comment:
KC_TRNS has already been pre-defined to be 7 spaces, you don't need to redefine it here.

Reply:
I did that to make it smaller to better align the layers how they were formatted, it just saved time.

Reply:
I just don't particularly like having it here because people tend to copy keymaps from other people and this kind of stuff proliferates pretty quick.

Reply:
Also, most of the core keycodes are 7 characters long, `KC_xxxx`.  So keeping them to that standard is preferred.

Reply:
So do you want to me change this or is it allowed to stay?

Reply:
You should remove it.

Reply:
Okay. :(